gluten-ut/spark34/src/test/scala/io/glutenproject/utils/velox/VeloxTestSettings.scala

```diff
@@ -943,6 +943,8 @@ class VeloxTestSettings extends BackendTestSettings {
     .exclude("SPARK-39557 INSERT INTO statements with tables with array defaults")
     .exclude("SPARK-39557 INSERT INTO statements with tables with struct defaults")
     .exclude("SPARK-39557 INSERT INTO statements with tables with map defaults")
+    // Field name must not be empty.
```

Comment:
Can you record the disable unit tests in https://github.com/oap-project/gluten/issues/3559? And I will pick them after this PR is merged. Thanks.

Reply:
sure, I disable native writer in *ParquetReadSchemaSuite because of same error, FYI.